### PR TITLE
✨Source OneSignal: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #28117: OneSignal

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source OneSignal to Low-Code CDK*
Closes #28117

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>
Full Test Results

image

</details>

## 🚨 User Impact 🚨
- No breaking changes, migration to low-code
<details><summary><strong>Migrate Connector: Source OneSignal</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

### CODE REVIEW

**Comment by btkcodedev** [comment]
Ps: All the tests are passing, Please ping via Slack for getting the credentials 
Slack ID: btkcodedev

image

**Review by sh4sh — COMMENTED** [review]
thanks for your work on this! is it possible to implement [pagination](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/pagination) for the streams that support it?

**Comment by btkcodedev** [comment]
@sh4sh Updated pagination, resolved comments, updated version
Test results - all passed in local
Please continue review :)

image

File: airbyte-integrations/connectors/source-onesignal/metadata.yaml [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-      enabled: true
-    oss:
-      enabled: true
+  releaseDate: 2023-08-12
   releaseStage: alpha
   supportLevel: community
   tags:
-    - language:python
+    - language:lowcode
+  supportLevel: certified
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-onesignal/metadata.yaml:25]:
```suggestion
```

Comment (sh4sh) [reply airbyte-integrations/connectors/source-onesignal/metadata.yaml:25]: Oops, meant to make this change on a different PR sorry! reverted in https://github.com/airbytehq/airbyte/pull/28941/commits/275446f72595f9413969c3d8af368748a087910f

**Review by sh4sh — APPROVED** [review]

**Comment by sh4sh** [comment]
Thank you for the contribution!

**Pull request merged**